File: dataset_reduzido/codenet/Java/Code/s998094687.java
import java.lang.Math.*;
import java.util.Scanner;
import java.util.Arrays;

class Main{
    public static void main(String[] args){
	final Scanner sc = new Scanner(System.in);
	int n=sc.nextInt();
	int[] a=new int[3]; 
	for(int i=0;i<n;i++){
	    a[0]=sc.nextInt();
	    a[1]=sc.nextInt();
	    a[2]=sc.nextInt();
	    Arrays.sort(a);	    
	    if(Math.pow(a[2],2)==Math.pow(a[0],2)+Math.pow(a[1],2)) System.out.println("YES");
	    else System.out.println("NO");
	}
    }
}

File: output/StarCoder-100/codenet/Java/Python/s998094687.py
import math

n = int(input())

for i in range(n):
    a = list(map(int, input().split()))
    a.sort()
    if (a[2] ** 2 == a[0] ** 2 + a[1] ** 2):
        print("YES")
    else:
        print("NO")
